src/sctools/platform.py

```diff
@@ -44,6 +47,44 @@ class GenericPlatform:
         merge multiple csr-format count matrices into a single csr matrix
     """
 
+    @classmethod
+    def tag_sort_bam(cls, args: Iterable=None) -> int:
+        """Command line entrypoint for sorting a bam file by one or more tags, followed by queryname.
+
+        Parameters
+        ----------
+        args : Iterable[str], optional
+            arguments list, for testing (see test/test_entrypoints.py for example). The default
+            value of None, when passed to `parser.parse_args` causes the parser to
+            read `sys.argv`
+
+        Returns
+        -------
+        return_call : 0
+            return call if the program completes successfully
+
+        """
+        parser = argparse.ArgumentParser()
+        parser.add_argument(
+            '-i', '--input_bam', required=True,
+            help='input bamfile')
+        parser.add_argument(
+            '-o', '--output_bam', required=True,
+            help='output bamfile')
+        parser.add_argument('-t', '--tags', nargs='+',
+                            help='tag(s) to sort by')
+        parser.set_defaults(raise_missing=True)
```

Comment:
Can you clarify what the `raise_missing` flag does here?

Reply:
Copy paste error - will remove.